src/loadBundle.js

```diff
@@ -0,0 +1,42 @@
+const { Base64 } = require('js-base64');
+const fs = require('fs');
+const logger = require('./helpers/logger');
+
+function loadELM(input) {
```

Comment:
this function should take in the bundle itself, not a file path to the bundle. So the `const bundle = JSON.parse(fs.readFileSync(input, 'utf8'));` should happen in the cli, and `bundle` should be the argument to this function